frontend/lib/src/components/shared/BaseColorPicker/styled-components.ts

```diff
@@ -38,6 +38,29 @@ export const StyledChromePicker = styled.div(({ theme }) => ({
   div: {
     fontFamily: `${theme.genericFonts.bodyFont} !important`,
   },
+  // The overrides below are a bit hacky but unfortunately the ChromePicker component
+  // doesn't offer a way to override labels and inputs, see
+  // https://github.com/casesandberg/react-color/issues/643
+  label: {
+    fontSize: `${theme.fontSizes.sm} !important`,
+    color: `${theme.colors.bodyText} !important`,
+  },
+  input: {
+    fontSize: `${theme.fontSizes.md} !important`,
+    height: `${theme.sizes.minElementHeight} !important`,
+    backgroundColor: `${theme.colors.secondaryBg} !important`,
+    color: `${theme.colors.bodyText} !important`,
+    borderRadius: `${theme.radii.default} !important`,
+    boxShadow: `none !important`,
+    "&:focus-visible": {
+      outline: `1px solid ${theme.colors.primary} !important`,
+    },
+  },
+  svg: {
+    // This is the arrow button on the right. The hover color for the background is
+    // apparently set by JS and not CSS, so we can't override it.
+    fill: `${theme.colors.bodyText} !important`,
```

Comment:
This adapts the style of the color picker dropdown elements to look like our widgets. I know this is a bit hacky but there's unfortunately no clean way to do this (see comment above). I'm not sure if we're OK with that much hackery but I guess the worst case is that this breaks and it looks like the default style of the ChromePicker component. 

Note that I also added a snapshot test for the color picker dropdown, so we should be able to catch when something breaks here.